jobsdb/jobsdb.go

```diff
@@ -652,6 +654,7 @@ func loadConfig() {
 	config.RegisterIntConfigVariable(10, &maxMigrateDSProbe, true, 1, "JobsDB.maxMigrateDSProbe")
 	config.RegisterInt64ConfigVariable(300, &maxTableSize, true, 1000000, "JobsDB.maxTableSizeInMB")
 	config.RegisterInt64ConfigVariable(1000, &backupRowsBatchSize, true, 1, "JobsDB.backupRowsBatchSize")
+	config.RegisterInt64ConfigVariable(64*bytesize.MB, &backupMaxTotalPayloadSize, true, 1, "JobsDB.maxBackupTotalPayloadSize")
```

Comment:
I think we should let the user provide the Input in MB and use the multiplication factor(4th parameter) to convert the integer provided to MB

Reply:
I don't believe this is necessary, but if we do any change on this one we should do the same change on all other config properties that deal with payload size limits

Reply:
Was just thinking about it from a usability perspective as adding an input as bytes might be a bit painful but it's not something which is blocking